Tests/Data/Elliptic/square_1x1_GroundWaterFlow/square_1e2_volumetricsourceterm.prj

```diff
@@ -2,6 +2,7 @@
 <OpenGeoSysProject>
     <meshes>
         <mesh>square_1x1_quad_1e2.vtu</mesh>
+        <mesh>square_1x1_quad_1e2_volumetric_source_term.vtu</mesh>
```

Comment:
This is still the entire mesh, isn't it? Is it special in any way? E.g., does it have an element property like `original_element_ids`? Btw. I didn't find code changes related to mapping d.o.f.s on the source term mesh to d.o.f.s on the whole domain. Does the current code work if the source term is applied to parts of the mesh only (in particular if the ST subset does not start from element ID 0)?

Reply:
Created test for strict subset.

Reply:
OK, thanks.